Makefile

```diff
@@ -38,15 +38,21 @@ BUILDCMD = go build -o $(TARGET) -v -ldflags "-X github.com/heptio/sonobuoy/pkg/
 BUILD = $(BUILDCMD) $(GOTARGET)/cmd/sonobuoy
 
 TESTARGS ?= -v -timeout 60s
-TEST = go test $(TEST_PKGS) $(TESTARGS)
 TEST_PKGS ?= $(GOTARGET)/cmd/... $(GOTARGET)/pkg/...
+TEST = go test $(TEST_PKGS) $(TESTARGS)
+
+
+# Vendor this someday
+INSTALL_GOLINT = go get -u github.com/golang/lint/golint
+GOLINT_FLAGS ?= -set_exit_status
+VET = go vet $(TEST_PKGS) && $(INSTALL_GOLINT) && golint $(GOLINT_FLAGS) $(TEST_PKGS)
```

Comment:
You could probably break this into 2 var.. 
VET 
LINT

Reply:
Could also add a gfmt check but now were entering into the whole we should break up the target b/c we're chaining too many things.